Source/WebCore/html/canvas/WebGLRenderingContextBase.cpp

```diff
@@ -1361,7 +1361,10 @@ bool WebGLRenderingContextBase::clearIfComposited(WebGLRenderingContextBase::Cle
                               m_colorMask[3] ? m_clearColor[3] : 0);
     } else
         m_context->clearColor(0, 0, 0, 0);
-    m_context->colorMask(true, true, true, true);
+    if (m_oesDrawBuffersIndexed)
```

Comment:
Is this hunk needed ?
I'm thinking the case 

set the color mask
enable the extension

vs

enable the extension
set the color mask


So these must end up in the same state inside ANGLE.
This should mean that  these must be interchangeable:

   colorMaskiOES(0, ...)
vs
   colorMask(...);

If they're interchangeable, then this hunk shouldn't be needed?